homeassistant/components/camera/diagnostics.py

```diff
@@ -0,0 +1,31 @@
+"""Diagnostics for camera."""
+
+from typing import Any
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import HomeAssistantError
+from homeassistant.helpers import entity_registry as er
+
+from . import _get_camera_from_entity_id
+from .const import DOMAIN
+
+
+async def async_get_platform_diagnostics(
```

Comment:
should this be called `async_get_diagnostics_for_entry`?

Reply:
I'm having trouble reasoning about that name given the existing one for integrations is `async_get_config_entry_diagnostics` so this is just re-ordered without `config` in it.

I can't tell if the intent is to have an on purpose different but similar name or you meant the existing function name.

The idea was that it was different to indicate it was a platform, not an integration with config entries, but happy to rename to something else as i'm less familiar with conventions.

Reply:
Actually, yeah why not call it the same? `async_get_config_entry_diagnostics`

Reply:
OK